docs/developers/connecting-to-celestia-ethermint.md

```diff
@@ -0,0 +1,30 @@
+# Connecting Metamask to Celestia Ethermint
+<!-- markdownlint-disable MD013 -->
+
+In this guide you'll connect your wallet to the Celestia Ethermint RPC Node.
+
+## Prerequisites
+
+If you do not have [MetaMask](https://metamask.io/) installed, please install it [here](https://metamask.io/download/)
+
+## Connecting MetaMask to Celestia Ethermint
+
+First, open the MetaMask settings and choose "Add Network"
+
+![Configuring MetaMask](/img/configure-metamask-1.png)
```

Comment:
Links to images on this page seem broken?

Reply:
might be an issue due to how Docusaurus finds the image path, here is how this looks locally
![Uploading Screen Shot 2022-06-20 at 6.39.07 PM.png…]()

Reply:
Ah I see. I think if you use a relative path from the file, it should work on both GitHub and Docusaurus. That would be preferred.